src/bitcoin-cli.cpp

```diff
@@ -867,6 +870,86 @@ static void GetWalletBalances(UniValue& result)
     result.pushKV("balances", balances);
 }
 
+/**
+ * ParseGetInfoResult takes in -getinfo result in UniValue object and parses it
+ * into a user friendly UniValue string to be printed on the console.
+ * @param[out] result  Reference to UniValue result containing the -getinfo output.
+ */
+static void ParseGetInfoResult(UniValue& result)
+{
+    if (!find_value(result, "error").isNull()) return;
+
+#ifndef WIN32
+    const std::string RESET = "\x1B[0m";
+    const std::string GREEN = "\x1B[32m";
+    const std::string BLUE = "\x1B[34m";
+    const std::string YELLOW = "\x1B[33m";
+    const std::string MAGENTA = "\x1B[35m";
+    const std::string CYAN = "\x1B[36m";
+#else
+    const std::string RESET = "";
+    const std::string GREEN = "";
+    const std::string BLUE = "";
+    const std::string YELLOW = "";
+    const std::string MAGENTA = "";
+    const std::string CYAN = "";
+#endif
```

Comment:
```suggestion
const std::string RESET = "\x1B[0m";
const std::string GREEN = "\x1B[32m";
const std::string BLUE = "\x1B[34m"; 
const std::string YELLOW = "\x1B[33m";
const std::string MAGENTA = "\x1B[35m"; 
const std::string CYAN = "\x1B[36m";
```

If you remove the code that ignores ANSI escape sequence on Windows, we can see colors on Windows in Command Prompt and PowerShell as well